aggie-coding-club/Rev-Registration
# Feedback form

## Description
Adds a feedback form accessible from the navbar. Note that even though we wanted to implement discord notifications, I'm waiting on Gannon to make that possible so we can just ignore it for now and do that in another PR or make the change to this once it's done.

## How to test
(Make sure you apply migrations before testing) click the button next to the login button on the navbar and try filling it with different ratings/comments, then check the DB to make sure they're submitted correctly

## Screenshots
|Before|After|
|--|--|
| ![image](https://user-images.githubusercontent.com/28655462/128640337-a5c5c48f-82d0-4117-9783-a3e714801cf8.png) | ![image](https://user-images.githubusercontent.com/28655462/128640324-a9e239ce-fcb7-4a76-b214-0e07784705b3.png) |

![image](https://user-images.githubusercontent.com/28655462/128640342-44b3065a-a7eb-44b5-b2a0-da89a0acc330.png)


## Related tasks
Closes #597.
